diff --git a/OpenSRC/APP/SNMP/Makefile b/OpenSRC/APP/SNMP/Makefile
index 5ecad4c..8120f8b 100755
--- a/OpenSRC/APP/SNMP/Makefile
+++ b/OpenSRC/APP/SNMP/Makefile
@@ -47,7 +47,7 @@ all:
 	cd $(SNMP_SRCDIR); \
 	patch -p1 < ../patch/$(SNMP_PKG_NAME)/0001-Remove-libpci.patch; \
 	./configure $(SNMP_CFG) $(SNMP_CFG_HOST) $(SNMP_CFG_FEATURE) ; \
-	make -j $(core); \
+	make; \
 	make install; fi;
 	$(Q)$(call BUILD_PACKAGE_BANNER,"Building SNMP END")
 
diff --git a/Utils/AOCTask.c b/Utils/AOCTask.c
index c623624..8fbb5d7 100644
--- a/Utils/AOCTask.c
+++ b/Utils/AOCTask.c
@@ -106,7 +106,7 @@ int main ()
         return STATUS_FAIL;
     }
 
-    //enablesmash();    // For debug FW using, it will enable without 0xf6 0xf9 command
+    enablesmash();    // For debug FW using, it will enable without 0xf6 0xf9 command
 
     while (1) {
         //b_i2c_access = PltPreAccessI2C (dw_dummy);
